packages/backend/src/services/ValidationService/ValidationService.ts

```diff
@@ -375,126 +374,122 @@ export class ValidationService extends BaseService {
     ): Promise<CreateDashboardValidation[]> {
         const existingFieldIds = existingFields.map(getItemId);
 
-        const dashboardSummaries = await this.dashboardModel.getAllByProject(
-            projectUuid,
-        );
-        const dashboards = await Promise.all(
-            dashboardSummaries.map((dashboardSummary) =>
-                this.dashboardModel.getById(dashboardSummary.uuid),
-            ),
-        );
```

Comment:
Main change in this file is replacing these lines with the new method `findDashboardsForValidation`. 
The rest is just adjusting variable names.